code/game/objects/items/robot/robot_upgrades.dm

```diff
@@ -751,6 +751,34 @@
 	if (BR)
 		R.model.remove_module(BR, TRUE)
 
+// Monkestation change: UwU-speak module for borgs because I hate borg players
+/obj/item/borg/upgrade/uwu
+	name = "cyborg UwU-speak 'upgrade'"
+	desc = "As if existence as an artificial being wasn't torment enough for the unit OR the crew."
+	icon_state = "cyborg_upgrade"
+
+/obj/item/borg/upgrade/uwu/action(mob/living/silicon/robot/R, user = usr)
+	. = ..()
+	RegisterSignal(R, COMSIG_MOB_SAY, PROC_REF(handle_speech))
+
+/obj/item/borg/upgrade/uwu/proc/handle_speech(datum/source, list/speech_args)
+	SIGNAL_HANDLER
+	var/message = speech_args[SPEECH_MESSAGE]
+
+	if(message[1] != "*")
+		message = replacetext(message, "ne", "nye")
+		message = replacetext(message, "nu", "nyu")
+		message = replacetext(message, "na", "nya")
+		message = replacetext(message, "no", "nyo")
+		message = replacetext(message, "ove", "uv")
+		message = replacetext(message, "r", "w")
+		message = replacetext(message, "l", "w")
+	speech_args[SPEECH_MESSAGE] = message
+
+/obj/item/borg/upgrade/uwu/deactivate(mob/living/silicon/robot/R, user = usr)
+	UnregisterSignal(R, COMSIG_MOB_SAY)
+//
```

Comment:
Could actually put this in it's own file inside the monkestation module folder

Reply:
Sure thing. I wasn't 100% sure what our stance was on putting new stuff in, but the location shouldn't matter as far as functionality goes, right? I saw we had some random stuff at the bottom of this file for reviving borgs or similar.

Reply:
Also, just to make sure I understand- you mean to take these changes from their code/game/<etc.> paths and put them into their own spot in monkestation/code/modules, correct? Should I go ahead and make a new directory for this PR, or is there a specific path you had in mind?

Reply:
Yeah. Just make new directories and files as needed to match

Reply:
Alright, I've moved everything over as requested, and cleaned up what needed cleaning up. Thanks for the clarification!